plugins/chat/assets/javascripts/discourse/components/chat-composer-upload.hbs

```diff
@@ -1,5 +1,11 @@
-<span class="chat-composer-upload">
-  <span class="preview">
+<div
+  class={{concat-class
+    "chat-composer-upload"
+    (if (eq this.type this.IMAGE_TYPE) "chat-composer-upload--image")
```

Comment:
Honestly we should just add an `isImage` getter to the JS, would make this less tedious in the HBS file